tests/chart_tests/test_byo_sa.py

```diff
@@ -253,9 +255,10 @@ def test_default_serviceaccount_names(template_name):
     pm_docs = [doc for doc in docs if doc["kind"] in pod_managers]
     service_accounts = [get_service_account_name_from_doc(doc) for doc in pm_docs]
     assert service_accounts
-    assert all((sa_name.startswith("release-name-") or sa_name == "default") for sa_name in service_accounts), (
-        f"Expected all service accounts to start with 'release-name-' but found {service_accounts} in {template_name}"
-    )
+    allowed_prefixes = ["release-name-", "default", "default-"]
+    assert all(
+        any(sa_name.startswith(prefix) or sa_name == prefix for prefix in allowed_prefixes) for sa_name in service_accounts
+    ), f"Expected all service accounts to start with 'release-name-' but found {service_accounts} in {template_name}"
```

Comment:
We can just go with this, which isn't any more risky:
```suggestion
    allowed_prefixes = ["release-name-", "default"]
    assert all(
        any(sa_name.startswith(prefix) for prefix in allowed_prefixes) for sa_name in service_accounts
    ), f"Expected all service accounts to start with a standard prefix but found {service_accounts} in {template_name}"
```